components/google-cloud/google_cloud_pipeline_components/experimental/__init__.py

```diff
@@ -63,12 +66,14 @@ def ForecastingValidationOp(**kwargs):
       tables and specs.
 
   Args:
-    **kwargs: Arbitrary keyword arguments.
+    **kwargs: See Accepted kwargs.
 
   Returns:
     None
   """
   # TODO(yzhaozh): update the documentation with Json object reference, example
   # and predefined validation requirements.
   return load_component_from_file(
-      './forecasting/validate/component.yaml')(**kwargs)
+      os.path.join(
+          os.path.dirname(__file__),
+          'forecasting/validate/component.yaml'))(**kwargs)
```

Comment:
Also can be covered in a follow up PR: I would suggest moving this code from `google_cloud_pipeline_components/experimental/__init__.py` to your projects specific `forecasting/__ini__.py` file.